ci-operator/README.md

```diff
@@ -183,19 +183,13 @@ complexity and duplication, which should be cleared in the future.
       - name: TEST_COMMAND
         value: "cp /tmp/admin.kubeconfig /tmp/admin.conf && make test-e2e"
       # required for jobs that use openshift-ansible to create a cluster
-      - name: RPM_REPO_BASEURL_REF
-        value: https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-3.10/.latest-rpms
+      - name: RPM_REPO
+        value: https://rpms.svc.ci.openshift.org/openshift-origin-v4.0/
```

Comment:
Could this be inferred from `tag_specification` and passed to templates by `ci-operator` automatically?